_includes/templates/iot-gateway/request-connector-basic-security-config.md

```diff
@@ -0,0 +1,21 @@
+
+One type of security configuration is basic.  
+For authorization will be used combination of username/password, provided in this section in config. 
```

Comment:
The client sends HTTP requests with the Authorization header that contains the word Basic word followed by a space and a base64-encoded string username:password.